compiler-plugin/src/main/kotlin/arrow/meta/quotes/classorobject/ObjectDeclaration.kt

```diff
@@ -0,0 +1,42 @@
+package arrow.meta.quotes.classorobject
+
+import arrow.meta.quotes.Scope
+import org.jetbrains.kotlin.psi.KtObjectDeclaration
+
+/**
+ * <code>""" object $name { $body }""".objectDeclaration</code>
+ *
+ * A template destructuring [Scope] for a [KtObjectDeclaration].
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.objectDeclaration
+ *
+ * val Meta.reformatObjectDeclaration: Plugin
+ *   get() =
+ *     "ReformatObjectDeclaration" {
+ *       meta(
+ *         objectDeclaration({ isObjectLiteral() }) { c ->
+ *           Transform.replace(
+ *             replacing = c,
+ *             newDeclaration = """
+ *               |$textOffset
+ *               |$`@annotations`
+ *               |$modality object $name $primaryConstructorParameterList$primaryConstructor: $superTypeListEntries {
+ *               |    $anonymousInitializers
+ *               |    $secondaryConstructor
+ *               |    $declarations
+ *               |}""".trimMargin().`object`
+ *             )
+ *           }
```

Comment:
@raulraja the only real difference that seemed worth picking out was the `offsetText`, but that was a very early part of the code and I don't know if it serves much purpose anymore.  

I also assume that the items for primary/secondary constructors will likely come back empty for object declarations but included them in there anyway (unless you want them removed?)

Cleaned up as much as I could that could be grabbed with methods that were able to be drilled down further (i.e. `$declarations` in lieu of `$classBody`)

Reply:
removed offsetText.